deploy/deploy_admin_terminal.yml

```diff
@@ -0,0 +1,9 @@
+# deploy_admin_terminal.yml
+---
+- hosts: admin_terminal
+  sudo: yes
+  roles:
+    - iptables
+    - common
+    - {role: python, when: "custom_repo != None"}
```

Comment:
What's it called when you use ansible to install ansible? ...anception?  Should that happen here?